input/resources/au-core-client.xml

```diff
@@ -81,7 +81,7 @@
           <blockquote>
```

Comment:
Remove support for http://hl7.org/fhir/uv/sdc/ImplementationGuide/hl7.fhir.uv.sdc
This comment not at the location of the content to be removed.

Reply:
Removed